<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerControl</class>
 <widget class="QWidget" name="PlayerControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayerControl</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="6">
    <widget class="QPushButton" name="fullscreenButton">
     <property name="text">
      <string>Full Screen</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSlider" name="volumeSlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="openButton">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="rateBox">
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="stopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="app_dpra.qrc">
       <normaloff>:/App_DPRA/icon-stop</normaloff>:/App_DPRA/icon-stop</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="playButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="app_dpra.qrc">
       <normaloff>:/App_DPRA/icon-play</normaloff>:/App_DPRA/icon-play</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="muteButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="app_dpra.qrc">
       <normaloff>:/App_DPRA/icon-vol</normaloff>:/App_DPRA/icon-vol</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>openButton</tabstop>
  <tabstop>playButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>muteButton</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>rateBox</tabstop>
  <tabstop>fullscreenButton</tabstop>
 </tabstops>
 <resources>
  <include location="app_dpra.qrc"/>
 </resources>
 <connections/>
</ui>
